core/domain/suggestion_services.py

```diff
@@ -593,12 +616,226 @@ def get_translation_suggestions_waiting_longest_for_review_per_lang(
+            replace_string = replace_string[:-1]
+        replace_string = replace_string.capitalize()
+        replace_string = ' [%s] ' % replace_string
+        return replace_string
+
+    # Replace all the opening <oppia-noninteractive-**> tags with their names
+    # capitalized in square brackets.
+    html_content_string_with_noninteractive_tag_replaced = re.sub(
+        r'<(oppia-noninteractive\s*?)[^>]+>', _replace_rte_tag,
+        html_content_string)
+    # Get rid of all of the other html tags, including closing tags and
+    # malformed html tags if they exist.
+    plain_text = html_cleaner.strip_html_tags(
+        html_content_string_with_noninteractive_tag_replaced
+    )
+    # Remove trailing and leading whitespace and ensure that all words are
+    # separated by a single space.
+    plain_text_without_contiguous_whitespace = ' '.join(plain_text.split())
+    return plain_text_without_contiguous_whitespace
+
+
+def create_reviewable_suggestion_email_info_from_suggestion(suggestion):
+    """Creates an object with the key information needed to notify reviewers or
+    admins that the given suggestion needs review.
+
+    Args:
+        suggestion: Suggestion. The suggestion used to create the
+            ReviewableSuggestionEmailInfo object. Note that the suggestion's
+            status must be in review.
+
+    Returns:
+        ReviewableSuggestionEmailInfo. The corresponding reviewable suggestion
+        email info.
+
+    Raises:
+        Exception. The keys in SUGGESTION_EMPHASIZED_TEXT_GETTER_FUNCTIONS
+            should be consist with the suggestion types offered on the
+            Contributor Dashboard.
+        Exception. The suggestion type must be offered on the Contributor
+            Dashboard.
+    """
+    # This check makes sure that an error is raised if a new suggestion type
+    # is added to the Contributor Dashboard but hasn't been added to
+    # SUGGESTION_EMPHASIZED_TEXT_GETTER_FUNCTIONS.
+    if set(suggestion_models.CONTRIBUTOR_DASHBOARD_SUGGESTION_TYPES) != set(
+            SUGGESTION_EMPHASIZED_TEXT_GETTER_FUNCTIONS.keys()):
+        raise Exception(
+            'Expected keys in SUGGESTION_EMPHASIZED_TEXT_GETTER_FUNCTIONS to '
+            'be [%s], received: [%s].' % (
+                ' '.join(
+                    suggestion_models.CONTRIBUTOR_DASHBOARD_SUGGESTION_TYPES),
+                ' '.join(SUGGESTION_EMPHASIZED_TEXT_GETTER_FUNCTIONS.keys())))
+
+    if suggestion.suggestion_type not in (
+            SUGGESTION_EMPHASIZED_TEXT_GETTER_FUNCTIONS):
+        raise Exception(
+            'Expected suggestion type to be offered on the Contributor '
+            'Dashboard, received: %s.' % suggestion.suggestion_type)
+
+    # Retrieve the html content that is emphasized on the Contributor Dashboard
+    # pages. This content is what stands out for each suggestion when a user
+    # views a list of suggestions.
+    get_html_content_for_emphasized_text = (
```

Comment:
Optional: I thought about the naming a bit more. How about `get_html_representing_suggestion`? Seems more direct since that's what this is for.

This is optional though, feel free not to do it if you prefer the existing name.

Reply:
Right, but in the context of this function, it's fairly clear what "representing" means, and this is a local variable within this function.

(I think my main concern was using "html", "content" and "text" all in the same variable name. We typically think of "text" as plaintext and "content" as the noninteractive part of an exploration card.)

Reply:
Fair enough, changed!

Reply:
Looks resolved.